src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  createDigits = () => {
+    const digits = [];
+    for (let i = 1; i < 10; i += 1) {
+      digits.push(
+        <button key={i} type="button">{i}</button>,
+      );
+    }
+    return digits;
+  };
+
+  render() {
+    return (
+      <div className="calculator">
+        <div className="display"> 0</div>
+
+        <div className="operators">
+          <button type="button">/</button>
+          <button type="button">*</button>
+          <button type="button">+</button>
+          <button type="button">-</button>
+
+          <button type="button">DEL</button>
+        </div>
+
+        <div className="digits">
+          { this.createDigits() }
+          <button type="button">0</button>
+          <button type="button">.</button>
+          <button type="button">=</button>
+        </div>
+
+      </div>
```

Comment:
- [ ] You've such a lovely design implementation in this component. However, since design at this stage is a strong project requirement, it's suggested that we implement this component as required. See screenshot below:

  ![image](https://user-images.githubusercontent.com/45002654/159137974-f84e8481-0d7c-49db-a52f-bea53b9d9954.png)
  
  
  Please consider addressing this.